### ISSUE TITLE
I'm excited about what you are doing

### ISSUE THREAD

**Issue by clscott** [ganezdragon/tree-sitter-perl#1]
Hi Ganezdragon,

I'm excited about your work on tree-sitter for perl.
Having these kinds of tools available is a real boon for perl.

Thank you!

**Comment by ganezdragon** [comment]
Ohh thank you!

Hoping for a beta release real soon :)

**Comment by FCO** [comment]
Is there a way of testing it with nvim?

**Comment by ganezdragon** [comment]
by nvim do you mean neovim? if so, it should work. I have a vscode client currently written here - https://github.com/ganezdragon/perl-language-server/tree/master/client

maybe this would help?

**Comment by FCO** [comment]
Yes, neovim... it's going ok until it tries to compile the code... it just says it could not compile. I'm following the tree-sitter steps for that... I'll try to take a look at your link and see I can find what I'm doing wrong. Thanks

**Comment by ganezdragon** [comment]
Okay.. but let me know if you wanted the `server` as a separate npm package. It might be needed when you are creating a client for neovim, and since the server won't have any changes.

**Comment by marcelarie** [comment]
Hi, really excited too for this parser!!

I'm a Nvim user too and I don't have any idea on how to setup external parser on Treesitter, but I will try.

@FCO did it work for you?
